### PR TITLE
Easter release week (release v8.16)

### PR DESCRIPTION
**Pull request by MichaIng**

### CODE REVIEW

File: docs/software/home_automation.md [hunk 1]
```diff
@@ -116,6 +116,9 @@ Domoticz is a Home Automation System that lets you monitor and configure various
 
     - HTTP: `http://<your.IP>:8124`
     - HTTPS: `https://<your.IP>:8424`
+    - Username: `admin`
+    - Password: `domoticz`
+    - NB: We recommend to create a new user with admin privileges via **Setup** -> **Users** and remove the above default one, or at least change the default password.
```
Comment (StephanStS) [inline docs/software/home_automation.md:121]: I always try to avoid these abbreviations and e.g. use "Remark:"

**Review by StephanStS — CHANGES_REQUESTED** [review]

File: docs/software/home_automation.md (hunk 1 above)
Comment (MichaIng) [reply docs/software/home_automation.md:121]: Ah, or a "tip" box.

Comment (MichaIng) [inline docs/software/home_automation.md:121]:
```suggestion

    !!! top "Change default user and password"
    We recommend to create a new user with admin privileges via **Setup** -> **Users** and remove the default one, or at least change the default password.
```

Comment (MichaIng) [reply docs/software/home_automation.md:121]: Like so? This is the green box with the flame:
```suggestion

    !!! tip "Change default user and password"
        We recommend to create a new user with admin privileges via **Setup** -> **Users** and remove the default one, or at least change the default password.
```

**Pull request merged**